ddtrace/contrib/internal/ray/__init__.py

```diff
@@ -0,0 +1,51 @@
+"""
+The ray integration traces:
+ - Job lifetime (job submit, job run)
+ - Task submission and execution
+ - Actor method submission and execution
+
+Enabling
+~~~~~~~~
+
+The Ray integration is enabled automatically when you use
+:ref:`ddtrace-run<ddtracerun>` or :ref:`import ddtrace.auto<ddtraceauto>`.
+
+Or use :func:`patch() <ddtrace.patch>` to manually enable the Ray integration::
+
+    from ddtrace import patch
+    patch(ray=True)
+
+The recommended way to instrument ray, is to instrument the ray cluster.
```

Comment:
Capitalizing the R in `Ray` since that's how they describe it in their documentation:
```suggestion
The recommended way to instrument Ray, is to instrument the Ray cluster.
```